packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts

```diff
@@ -14,22 +14,32 @@ function parseAndSanitize(description: string) {
   return parsedMarkdown;
 }
 
+function getParsedResults(eventTypes) {
```

Comment:
Let's type this correctly
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/3504472/186779120-94c4fad9-a3a5-41d6-8358-283516acbdb8.png">

Reply:
done